CHANGES.md

```diff
@@ -33,6 +46,19 @@ namespace, under `cylc.jinja.filters`.
 
 Cylc Review was also removed in this version.
 
+
+-------------------------------------------------------------------------------
+## __cylc-8.0a3 (2020-Q2?)__
+
+Fourth alpha release of Cylc 8.
+
+The following obsolete command line options have been removed:
+* `--host` (except for cylc run / restart)
+* `--owner`
+* `--port`
+* `--ssh-cylc`
+* `--no-login`
+
 -------------------------------------------------------------------------------
 ## __cylc-8.0a3 (2020-08?)__
 
```

Comment:
Small rebase error

Reply:
I'm not sure I see it - sorry - I'm being daft

Reply:
The heading `cylc-8.0a3` appears twice.

Reply:
Yes, in fact there seem to be a whole load of unwanted changes in this file - I hope that rebase hasn't been this flaky elsewhere. That the tests pass gives me much hope.